gossip/evm_state_reader.go

```diff
@@ -88,6 +90,13 @@ func (r *EvmStateReader) GetBlock(h common.Hash, n uint64) *evmcore.EvmBlock {
 }
 
 func (r *EvmStateReader) getBlock(h hash.Event, n idx.Block, readTxs bool) *evmcore.EvmBlock {
+	if n == 0 {
+		// fake genesis block for compatibility with web3
+		return evmcore.NewEvmBlock(evmcore.ToEvmHeader(&inter.Block{
+			Time:    inter.Timestamp(1608600000 * time.Second),
```

Comment:
nitpick: we can reuse `FakeGenesisTime` from **evmcore**
```suggestion
			Time:    evmcore.FakeGenesisTime,
```

Reply:
It would create a dependency cycle unfortunately

Reply:
I found that we already imported the **evmcore** package for `evmcore.NewEvmBlock`

Reply:
Oh yes, you're right